android_app/app/build.gradle

```
@@ -130,6 +130,8 @@ android {
 }
 
 dependencies {
+    implementation 'org.apache.commons:commons-text:1.9'
+    implementation 'org.apache.commons:commons-text:1.9'
```

This line should appear only once.
```suggestion
    implementation 'org.apache.commons:commons-text:1.9'
```